### PR TITLE
Independent versioning

### PR DESCRIPTION
**Pull request by libremente**
## Title
This PR introduces independent versioning as discussed in #39.
## Content
As of today, the only version key is the `publiccodeYmlVersion` key present in the `schema.rst` file. 
However, since the publiccode specifications allow to include country-specific extensions (defined in a dedicated schema file), it is deemed fundamental to version also those. 
As such, this PR introduces a new key called `version` to be inserted inside each country-specific schema. This changes also the tag/release approach previously used. 

@alranel: as discussed in #39, I am still not sure if `version` is an appropriate name since it seems way too generic to me. I made some name suggestions in that issue. 

## Review

- [x] Ensure your files are written following RST specs (*not MD!*)
- [x] Italian version
- [ ] English version
- [ ] Ask for review

### CODE REVIEW

File: docs/it/schema.it.rst [hunk 1]
```diff
@@ -4,8 +4,25 @@ Italia
...
+~~~~~~~~~~~~~~~~~~
+-  Tipo: stringa
+-  Presenza: obbligatoria
+-  Esempio: ``"1.0"``
+
+Questa chiave specifica la versione alla quale il presente schema di estensioni
+aderisce, per una questione di compatibilità diretta. Ogni Paese può avere una
+versione dello schema delle estensioni dedicato.
+
+**Nota Bene:** il valore di questa chiave è diverso da quello contenuto nella
```
Comment (alranel) [inline docs/it/schema.it.rst]:
```suggestion
**Nota Bene:** il valore di questa chiave è indipendente da quello contenuto nella
```

File: docs/it/schema.core.rst [hunk 2]
```diff
@@ -369,6 +371,7 @@ Un esempio per l’italiano:
 
    description:
      it:
+       version: ...
```
Comment (alranel) [inline docs/it/schema.core.rst]: Non va qui, va nell'elemento "it" top-level

File: docs/en/schema.core.rst [hunk 3]
```diff
@@ -350,6 +352,7 @@ An example for English:
 
    description:
      en:
+       version: ...
```
Comment (alranel) [inline docs/en/schema.core.rst]: Not here! Should go into the top-level "it" section

File: docs/it/schema.it.rst [hunk 4]
```diff
@@ -4,8 +4,25 @@ Italia
...
 
 Tutte le estensioni elencate qui di seguito sono specifiche per l'Italia e, di
-conseguenza, devono essere inserite in una sezione denominata con 
-il codice ``it``.
+conseguenza, devono essere inserite in una sezione denominata con il codice
+``it``. Tutti i Paesi sono specificati usando *country code* a due lettere
+seguendo lo standard ISO 3166-1 alpha-2.
+
+
+Chiave ``version``
```
Comment (libremente) [inline docs/it/schema.it.rst]:
```suggestion
Chiave ``CountryExtensionVersion``
```

**Comment by libremente** [comment]
Remember to tag `core:0.1/it:0.1` and release.

File: docs/it/schema.it.rst [hunk 5]
```diff
@@ -4,8 +4,25 @@ Italia
...
+
+
+Chiave ``countryExtensionVersion``
+~~~~~~~~~~~~~~~~~~
+-  Tipo: stringa
+-  Presenza: obbligatoria
+-  Esempio: ``"1.0"``
+
+Questa chiave specifica la versione alla quale il presente schema di estensioni
+aderisce, per una questione di compatibilità diretta. Ogni Paese può avere una
```
Comment (alranel) [inline docs/it/schema.it.rst]:
```suggestion
aderisce.
```

**Pull request merged**